src/fixit/api.py

```diff
@@ -17,6 +19,27 @@
 logger = logging.getLogger(__name__)
 
 
+def print_result(result: Result, debug: bool) -> None:
```

Comment:
Set a default value for `debug = False`, to make this a nicer API for external usage.

Would also appreciate a docstring here, and add it to the "simple API" sections of the sphinx docs (see `docs/api.rst`).